lib/sidekiq_monitoring_bitcrowd.rb

```diff
@@ -9,4 +9,12 @@ module SidekiqMonitoring
 
   mattr_accessor :http_auth_name
   mattr_accessor :http_auth_password
+
+  def self.redis
+    if SidekiqMonitoring.redis_url.present?
```

Comment:
Actually not sure if we should memoize this connection here... any input on that?